docs/devops/hardfork.md

```diff
@@ -0,0 +1,164 @@
+# Hardfork
+
+## introduction
+
+Currently, Axon has only one hardfork named `Andromeda`. The name of the hardfork is derived from one of the 88 constellations, and in the future, names will continue to be chosen based on the [order of the constellations](https://en.wikipedia.org/wiki/IAU_designated_constellations#List). After enabling `Andromeda`, [validators](https://github.com/axonweb3/axon/blob/f9974e62924693494476560316db9f70bc650b80/devtools/chain/nodes/node_1.toml#L3) are allowed to modify metadata, such as `max_contract_limit`, through [the system contract `0xffffffffffffffffffffffffffffffffffffff01`](https://docs.axonweb3.io/contract/system_contacts#metadata). Axon already has [hardfork test CI](https://github.com/axonweb3/axon/blob/f9974e62924693494476560316db9f70bc650b80/.github/workflows/hardfork_test.yml)  and [a test project available](https://github.com/axonweb3/axon-hardfork-test) for your reference.
```

Comment:
There is ambiguity. If hardfork is not activated, only the max contract limit cannot be modified. Other configs can be modified.